src/gui/mainwindow.h

```diff
@@ -175,6 +175,9 @@ private slots:
     void on_actionOpen_triggered();
     void on_actionDownloadFromURL_triggered();
     void on_actionExit_triggered();
+#ifdef Q_OS_MAC
```

Comment:
Please move this to the bottom of `private slots:` group, everything else is good.

Reply:
Are you sure? Below is the group of unrelated functions, and currently it is in `on_action` group.
Or did you mean to put it below on_actionLock_triggered?

Reply:
>Are you sure? 

Yes, because I find `#if` conditionals quite disturbing when reading code and grouping them together minimize this disturbance.

Reply:
Well, ok, done.